index.html

```diff
@@ -61,5 +61,203 @@ <h1>
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+      <article class="portfolio-item">
+        <img src="./assets/images/tonic2.png" alt="tonic" class="mt-12" />
+        <div class="title-block mt-12">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-role">
+            <p class="client">CANOPY</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="role">Back End Dev</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+      <article class="portfolio-item">
+        <img src="./assets/images/tonic.png" alt="tonic" class="mt-12" />
+        <div class="title-block mt-12">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-role">
+            <p class="client">Multi-Post Stories</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="role">Back End Dev</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+    </section>
+
+    <section class="about-section">
+      <div class="about-top">
+        <h2 class="about-primary-text">About<br />Myself</h2>
```

Comment:
- [ ] The top-right corner of the about section is curved too and the background color above the about section is different from the background color of the about section. Kindly implement these features so as to improve your user interface 💯 
- Your design:
![Capture2](https://user-images.githubusercontent.com/85113567/154784340-7a58d01e-4232-49e2-b083-5c98ff820a7f.PNG)
- Figma design:

![Capture6](https://user-images.githubusercontent.com/85113567/154784366-28f96db4-87a7-487c-9b1c-4826ffe6721b.PNG)